index.js

```diff
@@ -0,0 +1,144 @@
+const header = document.createElement('header');
+
+const headerh1 = document.createElement('h1');
+headerh1.setAttribute('id', 'logo');
+headerh1.innerText = 'Techno Mobile';
+
+const button = document.createElement('button');
+button.setAttribute('type', 'button');
+button.innerHTML = '<i class="fas fa-bars"></i>';
```

Comment:
- [ ] Your menu button should have a hamburger menu icon. 

Usage of three horizontal lines for mobile menu buttons became a convention nowadays. If you want to make your application easier to use and accessible. 
I can see you added a font-awesome icon but you don't have font configuration it doesn't work.